### PR TITLE
Support IE11 - finally done!

### PR DESCRIPTION
**Pull request by suro1995**
This change finally get rewiremock to work with IE11!

### The changes are:
1. Replace Array.prototype.find
    * Use Array.prototype.some to check element existance
    * Use Array.prototype.filter combined with Array.prototype.unshift to get the element

2. Add babel-plugin-transform-runtime to transform Promise and Object.assign
(thus babel-plugin-transform-object-assign is not needed)

3. Replace string.prototype.endsWith with a custom helper function

### Verification
Pass in local IE tests!

### CODE REVIEW

**Comment by suro1995** [comment]
What does the first failure actually means?

File: webpack/src/module.js [hunk 1]
```diff
@@ -13,9 +13,9 @@ var Module = {
       .sort(function(a, b) { return a.length - b.length; });
     var targetFileIndex = targetFile + '/index';
 
-    var asIs = keys.find(function(name) { return name.indexOf(fileName) >= 0; });
-    var asFile = keys.find(function(name) { return name.indexOf(targetFile) >= 0; });
-    var asIndex = keys.find(function(name) { return name.indexOf(targetFileIndex) >= 0; });
+    var asIs = keys.filter(function(name) { return name.indexOf(fileName) >= 0; }).shift();
```
Comment (theKashey) [inline webpack/src/module.js]: Probably it worth in future to convert this to a single cycle, for performance sake.

**Comment by theKashey** [comment]
> What does the codecov/patch failure actually means?
It complaining that you have decreased code coverage, while you didn't.

**Comment by theKashey** [comment]
Released

File: webpack/src/module.js (hunk 1 above)
Comment (suro1995) [reply webpack/src/module.js]: Yes, maybe create a module like utils.js or something.

**Pull request merged**